src/routes/(admin)/tests/test-[type]/[id]/QuestionList.svelte

```
@@ -1,15 +1,23 @@
+<script lang="ts">
 	import Button from '$lib/components/ui/button/button.svelte';
-	import QuestionDialog from './QuestionDialog.svelte';
-	import { columns } from './question_table/columns.js';
-	import GripVertical from '@lucide/svelte/icons/grip-vertical';
-	import Trash2 from '@lucide/svelte/icons/trash-2';
-	import Eye from '@lucide/svelte/icons/eye';
-	let { formData, questions } = $props();
+	import QuestionSelectionDialog from './question-selection/QuestionSelectionDialog.svelte';
+	import SelectedQuestionsList from './question-selection/SelectedQuestionsList.svelte';
+
+	let { formData, questions, questionParams, selectedQuestions = [] } = $props();
```

ESLint

[error] 'selectedQuestions' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)